test/unittests/parser_test.cpp

```diff
@@ -1054,9 +1061,9 @@ TEST(parser, interleaved_custom_section)
     const auto type_section = make_vec({functype_void_to_void});
     const auto func_section = make_vec({"00"_bytes});
     const auto code_section = make_vec({add_size_prefix("000b"_bytes)});
-    const auto bin = bytes{wasm_prefix} + make_section(0, bytes{4}) +
```

Comment:
These were actually valid.

Reply:
They don't look valid to me. You promised a vector of 4 bytes without providing any actual bytes.

Reply:
I mean custom sections with `n` zero bytes is valid.

Reply:
Oh, I see. This is one of the C++11 pitfalls.
The `bytes{4}` creates `"04"` because the constructor for `std::initializer_list` has priority if `{}` are used.
You meant `bytes(4)` to create `"00000000"`. To be fully precise such constructor does not exist (opposite to `std::vector`) - the zero must be explicit here: `bytes(4, 0)`.

Reply:
I can convert it to what is was meant to be.

Reply:
It doesn't really matter for this test case. We should however add a test case which includes unparsed data (and not just a parsed name) in the custom section.

Reply:
Pushed this test case here.